app/scheduling/directives/termCalendar.js

```diff
@@ -104,34 +104,104 @@ schedulingApp.directive("termCalendar", this.termCalendar = function ($rootScope
 				});
 			};
 
+			// supply a color and amount to shift it by
+			// example to lighten: lightenOrDarkenColor("#F06D06", 20);
+			// example to darken: lightenOrDarkenColor("#F06D06", -20);
+			var lightenOrDarkenColor = function(col, amt) {
+				var usePound = false;
+
+				if (col[0] == "#") {
+					col = col.slice(1);
+					usePound = true;
+				}
+
+				var num = parseInt(col,16);
+				var r = (num >> 16) + amt;
+
+				if (r > 255) {
+					r = 255;
+				} else if (r < 0) {
+					r = 0;
+				}
+
+				var b = ((num >> 8) & 0x00FF) + amt;
+
+				if (b > 255) {
+					b = 255;
+				} else if (b < 0) { b = 0; }
+
+				var g = (num & 0x0000FF) + amt;
+
+				if (g > 255) {
+					g = 255;
+				} else if (g < 0) {
+					g = 0;
+				}
+
+				return (usePound ? "#" : "") + (g | (b << 8) | (r << 16)).toString(16);
+			};
+
 			var getActivities = function () {
 				// Each of these If blocks will add to a 'events array'
 				// The event making function will color them appropriately
 				var calendarActivities = [];
 
 				// Add Selected sectionGroup activities
 				if (scope.view.state.uiState.selectedSectionGroupId) {
-					var unstyledEvents = sectionGroupToActivityEvents(scope.view.state.sectionGroups.list[scope.view.state.uiState.selectedSectionGroupId]);
+					var sectionGroup = scope.view.state.sectionGroups.list[scope.view.state.uiState.selectedSectionGroupId];
+					var unstyledEvents = sectionGroupToActivityEvents(sectionGroup);
+					var tagColor = calculateTagColor(sectionGroup);
+
+					var textColor = tagColor ? "#FFFFFF" : activeEventTextColor;
```

Comment:
Define a variable for whatever "#FFFFFF" stands for, e.g. inactiveEventTextColor ?